pkg/webhook/mutation.go

```diff
@@ -162,9 +166,47 @@ func (h *mutationHandler) Handle(ctx context.Context, req admission.Request) adm
 }
 
 func (h *mutationHandler) mutateRequest(ctx context.Context, req admission.Request) (admission.Response, error) {
-	// TODO: place mutation logic here
-	mutatedObject := req.Object
-	resp := admission.PatchResponseFromRaw(req.Object.Raw, mutatedObject.Raw)
+	okResp := admission.Response{
+		AdmissionResponse: admissionv1beta1.AdmissionResponse{
+			Allowed: true,
+			Result: &metav1.Status{
+				Code: int32(http.StatusOK),
+			},
+		},
+	}
+	ns := &corev1.Namespace{}
+	if err := h.client.Get(ctx, types.NamespacedName{Name: req.AdmissionRequest.Namespace}, ns); err != nil {
```

Comment:
Should we handle the special case where the object is a namespace itself?

Reply:
+1

Also, we should only gather the namespace once per request in order to avoid skew due to namespace labels changing mid-evaluation.

Reply:
(oops, looks like we're doing that once-per-request thing here already)

Reply:
We still need to handle this special case where the object itself is a namespace

Reply:
Done under @mmirecki 's name :-)